aesthetic_ants/penguin/utils.py

```diff
@@ -0,0 +1,19 @@
+# Contains utility functions
+import math
+import pyglet
+
+
+def angle_between(x1: float, y1: float, x2: float, y2: float) -> float:
+    """
+    Returns the angle between the coordinates (x1, y1) and (x2, y2) in radians
+    """
+    dx = x2 - x1
+    dy = y2 - y1
+
+    # We return negative because pyglet and math treat rotation differently
+    return -math.atan2(dy, dx)
+
+
+loader = pyglet.resource.Loader(path="../resources")
+
+keys = pyglet.window.key.KeyStateHandler()
```

Comment:
Curious why keys is global/module-level. You don't need to change it, I'm just curious.

Reply:
Keys is global so that any module can simply import keys from utils and test it during their update step

Reply:
Fair enough. I think only the player would need it for now but this should be ok.